tests/commands/conftest.py

```diff
@@ -7,7 +7,7 @@
 
 
 @pytest.fixture()
-def config():
+def mock_config():
```

Comment:
We're not mocking a config here. We're providing a sample config instead. `mock_config` is inaccurate. I probably would try `sample_config` or something similiar